### ISSUE TITLE
Inside the ocean

### ISSUE THREAD

**Issue by FanRenLa** [Roadhog360/Et-Futurum-Requiem#173]
**Starting Questions**
- [ ] This is a mod incompatibility, if I use this in vanilla Forge, with only Et Futurum Requiem installed it works as I'd expect it to.

_Please specify the actual version... please don't just say "latest", tell me the version number of the mod in-game, or the name the jar had when downloading it._
Version:2.4.0
If this is a crash, please upload the report:

**Describe the bug**
_A clear and concise description of what the bug is._
Inside the ocean
**To Reproduce**
1. Wax a copper block of any kind
2. Using any compatible axe item, right click to scrape it off
3. The game crashes

**Mod Incompatibilities**
_If this was a mod incompatibility, what are the possible suspects? If you're not 100% sure, you can still list your suspected mods here, just tell me if you're not certain they actually cause it._

**Mod List**
_If you have posted a crash report above or this is not a mod incompatibility, you can skip this. If you're filling this out because of a mod incompatibility that isn't a crash, crash reports generate a mods list you can copy. Please use Pastebin if it is large. If you don't have an up-to-date crash report, you can hold F3 + C for 10 seconds while in a world to crash the game, which will give you a crash report._

**Screenshots**

**Additional context**
Any other info I may have missed. If you know the code which causes the error please post a link to it.
![2022-08-31_13 59 54](https://user-images.githubusercontent.com/83486178/187606209-6e824adb-c6d3-4e3e-a99e-95e2f546b3b0.png)
![2022-08-31_13 59 40](https://user-images.githubusercontent.com/83486178/187606218-1d27e181-25bf-4c96-88bd-94056e7fc025.png)

**Comment by Roadhog360** [comment]
No information, duplicate of #149.

Please fill out the template next time instead of ignoring it. It exists for a reason. Read the pinned issue

**Comment by GregoriusT** [comment]
But the Template looks very filled out to me. Clearly the process of filling out a template should not involve erasing and replacing parts of it. Also stop expecting people that dont even know English at all, to be able to understand your template. Anyone using Google Translate, will be absolutely unable to use your Template.

**Comment by Roadhog360** [comment]
90% of the time we are able to communicate just fine through translation, the template is written in the same exact English. If anything is unclear in the template, nothing is stopping anyone from asking for clarification, which I would be happy to provide

I don't really know how else I'm supposed to ask questions without typing it directly into the template. The only other way is making this large and complicated xml document or whatever with weird syntax I don't understand in order to enforce a proper template, because for some reason GitHub provides no GUI for it...

**Comment by GregoriusT** [comment]
Just remove the "example answers" from your Questions, those are the primary issue, heck they even hurt your readability in the end, since you have to read through all the leftover "example answers" every time someone posts an Issue.

Also a lot of Issues tend to not have the "step 1 2 3 to reproduce" thing, because it is not applicable to most issues.

Lemme draft up a better Template, that asks basically the same questions... (totally didnt do that on github before, sooooo might take a bit XD)

**Comment by Roadhog360** [comment]
Fair, I guess the example for reproduction wasn't really necessary at all now that I think about it

**Comment by makamys** [comment]
I've got some changes to the template that I can suggest, I made a PR (#174) with them to avoid a revert war.

**Comment by GregoriusT** [comment]
Lol I intentionally made a separate template to avoid such a thing, that way all the ways exist at once. ;)

But yeah I clearly do not know much about how that System works, because the [ ] Boxes dont really "click" well.

**Comment by makamys** [comment]
I really don't think we should have two templates that are identical aside from formatting, that's extremely counter-intuitive.

You can only click checkboxes after the post has been made, not while editing it. I added a hint about how to do this to the template now.

**Comment by Roadhog360** [comment]
> You can only click checkboxes after the post has been made, not while editing it

You can also put an x in it while still making the post or editing it. Although I don't really expect people to know that either, I really just expect anything indicating the answer, even if it's not really in the box

**Comment by GregoriusT** [comment]
Oh so that is how the boxes work.

And I figured once all improvements and criticisms are done, one of the templates could be removed.

When making my Template the only two really important things for me were
"put 5 empty lines between each question" to make it obvious you should type your text there (since github removes empty lines upon posting, this is invisible in the end product),
and the other point was avoiding to tell the Issue Poster to remove lines that do not apply, because this just causes confusion. So I instead gave hints in the Question to describe the Issue that things like Screenshots exist.

But eh, I'm still learning how to do it, weirdly enough people on my Mod can use a blank Issue template perfectly fine. Though maybe  I should tell them how to do the whole Log Files thing. Btw I think Forge itself is what removed the fml-client-latest.log and not the Launchers, so that is something I need to deal with soon too, because freshly downloaded Forge Installer installs do not seem to have those Logs...

**Comment by makamys** [comment]
> You can also put an x in it while still making the post or editing it.

Yeah, [that's exactly what the hint I added says](https://github.com/Roadhog360/Et-Futurum-Requiem/pull/174/commits/7198abd57ea7572487bf91db8568feacfc287f36).

> And I figured once all improvements and criticisms are done, one of the templates could be removed.

Creating a branch with the new template and merging it into master once it's finished would have been better, since now the repo will be left in a weird state until the template is finished. But no big deal either way.

Also yeah, I agree with your changes, I just cleaned them up a bit.
